testsuite/MDAnalysisTests/analysis/test_hole2.py

```diff
@@ -98,6 +98,23 @@ def test_symlink_file(self, tmpdir):
             assert not fixed.endswith(long_name)
 
 
+class TestOserror(object):
+    u = mda.Universe(MULTIPDB_HOLE)
+
+    errmsg1 = exe_err.format(name='dummy_path', kw='executable')
```

Comment:
In this case it would be clearer for the error messages to be defined within the individual test functions, there's really no need for these to be defined by the whole class.